src/runtime-rs/crates/hypervisor/src/qemu/cmdline_generator.rs

```diff
@@ -884,6 +884,57 @@ impl ToQemuParams for NetDevice {
     }
 }
 
+#[derive(Debug)]
+struct SerialDevice {
```

Comment:
As per the naming convention of this file (mentioned in another comment on this PR), this should be called `DeviceVirtioSerial`.

Reply:
Renamed. Thanks!